<ui version="4.0" >
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>627</width>
    <height>683</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <layout class="QHBoxLayout" >
    <item>
     <widget class="QGroupBox" name="groupBox" >
      <property name="sizePolicy" >
       <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
        <horstretch>1</horstretch>
        <verstretch>1</verstretch>
       </sizepolicy>
      </property>
      <property name="title" >
       <string>Generators</string>
      </property>
      <layout class="QVBoxLayout" >
       <property name="leftMargin" >
        <number>4</number>
       </property>
       <property name="topMargin" >
        <number>4</number>
       </property>
       <property name="rightMargin" >
        <number>4</number>
       </property>
       <property name="bottomMargin" >
        <number>4</number>
       </property>
       <item>
        <widget class="QListView" name="generators_" />
       </item>
       <item>
        <layout class="QHBoxLayout" >
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" >
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="addGenerator_" >
           <property name="text" >
            <string>Add</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="removeGenerator_" >
           <property name="enabled" >
            <bool>false</bool>
           </property>
           <property name="text" >
            <string>Delete</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" >
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" >
      <item>
       <widget class="QGroupBox" name="groupBox_2" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="title" >
         <string>Message Format</string>
        </property>
        <layout class="QVBoxLayout" >
         <item>
          <layout class="QHBoxLayout" >
           <item>
            <widget class="QLabel" name="label_3" >
             <property name="text" >
              <string>Sample Freq</string>
             </property>
             <property name="alignment" >
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="sampleFrequency_" >
          <property name="keyboardTracking" >
           <bool>false</bool>
          </property>
             <property name="minimum" >
              <double>1.000000000000000</double>
             </property>
             <property name="maximum" >
              <double>9999.000000000000000</double>
             </property>
             <property name="value" >
              <double>1.000000000000000</double>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="sampleFrequencyScale_" >
             <property name="sizePolicy" >
              <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="currentIndex" >
              <number>2</number>
             </property>
             <item>
              <property name="text" >
               <string>Hz</string>
              </property>
             </item>
             <item>
              <property name="text" >
               <string>kHz</string>
              </property>
             </item>
             <item>
              <property name="text" >
               <string>MHz</string>
              </property>
             </item>
             <item>
              <property name="text" >
               <string>GHz</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" >
           <item>
            <widget class="QLabel" name="label_8" >
             <property name="text" >
              <string>Range Min/Max</string>
             </property>
             <property name="alignment" >
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="rangeMin_" >
          <property name="keyboardTracking" >
           <bool>false</bool>
          </property>
             <property name="maximum" >
              <double>1000.000000000000000</double>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="rangeMax_" >
          <property name="keyboardTracking" >
           <bool>false</bool>
          </property>
             <property name="minimum" >
              <double>1.000000000000000</double>
             </property>
             <property name="maximum" >
              <double>1000.000000000000000</double>
             </property>
             <property name="value" >
              <double>300.000000000000000</double>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" >
           <item>
            <widget class="QLabel" name="label_9" >
             <property name="text" >
              <string>Value Min/Max</string>
             </property>
             <property name="alignment" >
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="valueMin_" >
          <property name="keyboardTracking" >
           <bool>false</bool>
          </property>
             <property name="minimum" >
              <number>-65535</number>
             </property>
             <property name="maximum" >
              <number>65535</number>
             </property>
             <property name="value" >
              <number>-8192</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="valueMax_" >
          <property name="keyboardTracking" >
           <bool>false</bool>
          </property>
             <property name="minimum" >
              <number>-65535</number>
             </property>
             <property name="maximum" >
              <number>65535</number>
             </property>
             <property name="value" >
              <number>8192</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" >
           <item>
            <widget class="QLabel" name="label_6" >
             <property name="text" >
              <string>Samples / Message</string>
             </property>
             <property name="alignment" >
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="sampleCount_" >
          <property name="keyboardTracking" >
           <bool>false</bool>
          </property>
             <property name="minimum" >
              <number>100</number>
             </property>
             <property name="maximum" >
              <number>10000</number>
             </property>
             <property name="value" >
              <number>2048</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" >
           <item>
            <widget class="QLabel" name="label_7" >
             <property name="text" >
              <string>Messages / Rotation</string>
             </property>
             <property name="alignment" >
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="radialCount_" >
          <property name="keyboardTracking" >
           <bool>false</bool>
          </property>
             <property name="minimum" >
              <number>1</number>
             </property>
             <property name="maximum" >
              <number>10000</number>
             </property>
             <property name="value" >
              <number>5000</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" >
           <item>
            <spacer>
             <property name="orientation" >
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" >
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QCheckBox" name="doComplex_" >
             <property name="text" >
              <string>Complex Data</string>
             </property>
             <property name="checked" >
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <spacer>
             <property name="orientation" >
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" >
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="emitter_" >
        <property name="title" >
         <string>Publisher</string>
        </property>
        <layout class="QGridLayout" >
         <item row="3" column="0" >
          <widget class="QLabel" name="label_33" >
           <property name="text" >
            <string>Connections:</string>
           </property>
           <property name="alignment" >
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="2" column="0" >
          <widget class="QLabel" name="label_5" >
           <property name="text" >
            <string>Multicast Address:</string>
           </property>
           <property name="alignment" >
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="0" >
          <widget class="QLabel" name="label_4" >
           <property name="text" >
            <string>Connection Type:</string>
           </property>
           <property name="alignment" >
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="0" column="0" >
          <widget class="QLabel" name="label_2" >
           <property name="sizePolicy" >
            <sizepolicy vsizetype="Preferred" hsizetype="Minimum" >
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text" >
            <string>Name:</string>
           </property>
           <property name="alignment" >
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="0" column="1" >
          <widget class="QLineEdit" name="name_" >
           <property name="sizePolicy" >
            <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
             <horstretch>1</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text" >
            <string>SignalGenerator</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1" >
          <widget class="QLabel" name="connections_" >
           <property name="text" >
            <string>0</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1" >
          <widget class="QLineEdit" name="address_" >
           <property name="sizePolicy" >
            <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
             <horstretch>1</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text" >
            <string>237.1.2.110</string>
           </property>
           <property name="maxLength" >
            <number>15</number>
           </property>
          </widget>
         </item>
         <item row="1" column="1" >
          <widget class="QComboBox" name="connectionType_" >
           <property name="sizePolicy" >
            <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
             <horstretch>1</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="currentIndex" >
            <number>1</number>
           </property>
           <item>
            <property name="text" >
             <string>TCP</string>
            </property>
           </item>
           <item>
            <property name="text" >
             <string>Multicast</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_3" >
        <property name="title" >
         <string>Emitter</string>
        </property>
        <layout class="QVBoxLayout" >
         <property name="leftMargin" >
          <number>4</number>
         </property>
         <property name="topMargin" >
          <number>4</number>
         </property>
         <property name="rightMargin" >
          <number>4</number>
         </property>
         <property name="bottomMargin" >
          <number>4</number>
         </property>
         <item>
          <layout class="QHBoxLayout" >
           <item>
            <widget class="QLabel" name="label_10" >
             <property name="text" >
              <string>Message Count</string>
             </property>
             <property name="alignment" >
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="messageCount_" >
          <property name="keyboardTracking" >
           <bool>false</bool>
          </property>
             <property name="minimum" >
              <number>1</number>
             </property>
             <property name="maximum" >
              <number>99999</number>
             </property>
             <property name="value" >
              <number>100</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" >
           <item>
            <widget class="QLabel" name="label" >
             <property name="text" >
              <string>Emitter Frequency</string>
             </property>
             <property name="alignment" >
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="emitterFrequency_" >
          <property name="keyboardTracking" >
           <bool>false</bool>
          </property>
             <property name="minimum" >
              <number>1</number>
             </property>
             <property name="maximum" >
              <number>9999</number>
             </property>
             <property name="value" >
              <number>180</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="emitterFrequencyScale_" >
             <property name="sizePolicy" >
              <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="currentIndex" >
              <number>0</number>
             </property>
             <item>
              <property name="text" >
               <string>Hz</string>
              </property>
             </item>
             <item>
              <property name="text" >
               <string>kHz</string>
              </property>
             </item>
             <item>
              <property name="text" >
               <string>MHz</string>
              </property>
             </item>
             <item>
              <property name="text" >
               <string>GHz</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" >
           <item>
            <spacer>
             <property name="orientation" >
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" >
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="startStop_" >
             <property name="sizePolicy" >
              <sizepolicy vsizetype="Fixed" hsizetype="Minimum" >
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text" >
              <string>Start</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="rewind_" >
             <property name="text" >
              <string>Rewind</string>
             </property>
             <property name="shortcut" >
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <spacer>
             <property name="orientation" >
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" >
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" >
            <size>
             <width>306</width>
             <height>16</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>627</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar" />
 </widget>
 <resources/>
 <connections/>
</ui>
